tests/responses/ok-test.js

```diff
@@ -6,6 +6,10 @@ var path = require('path');
 var Sails = require('sails').Sails;
 var test = require('tape-catch');
 var loadConfig = require('../helpers/load-config');
+var jsonApiSchema = JSON.parse(
+      require('fs').readFileSync('tests/jsonapi.schema', 'utf8')
+    );
+var validateJsonApi = require('ajv')().compile(jsonApiSchema);
```

Comment:
Would you mind moving the `require` to the top of the file, and creating a variable for `require('ajv')()`?  Really just a style preference, but I like to keep dependencies at the top of a file for clarity.